### PR TITLE
Added bids ttl value to the external config file #101

### PR DESCRIPTION
**Pull request by griffobeid**

### CODE REVIEW

File: server/store/bids.js [hunk 1]
```diff
@@ -22,13 +23,13 @@ const saveBid = async ({ vehicle_id, price, time_to_pickup, time_to_dropoff }, r
   );
 
   // Set TTL for bid
-  redis.expire(`bids:${bidId}`, 3600);
+  redis.expire(`bids:${bidId}`, config.bids_ttl);
```
Comment (TalAter) [inline server/store/bids.js]: I believe these three need to be `config('bids_ttl')`

File: server/store/bids.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 const redis = require('./redis');
+const config = require('./config');
```
Comment (TalAter) [inline server/store/bids.js]: I think the correct path to the file would be `'../config'`

**Comment by griffobeid** [comment]
Okay, fixed

**Comment by TalAter** [comment]
Thank you so much @griffobeid 

And congratulations on doing your first open source pull request 🍾 

griffobeid__griffin_obeid_

Please join us in our [gitter channel](https://gitter.im/DAVFoundation/DAV-Contributors) where we discuss DAV, and more opportunities to get involved with open source :octocat:

**Pull request merged**